### ISSUE TITLE
Go to label in GMail does not read correctly in Chrome

### ISSUE THREAD

**Issue by mjanusauskas** [nvaccess/nvda#6075]
Using Chrome Canary with the standard GMail UI:
Press G then L to go to a label.
Start to type a label name that is auto-completed.
Note that NVDA speaks "unknown" and does not read the control autocomplete or results when pressing down arrow.
This works correctly in Firefox with NVDA.

**Comment by jcsteh** [comment]
This is due to a bug in Chrome. Filed [Chromium issue 620432](https://bugs.chromium.org/p/chromium/issues/detail?id=620432).

**Comment by mjanusauskas** [comment]
Thank you for that.

**Comment by jcsteh** [comment]
Fixed in Chrome.
